lib/mbta_v3_api/repository.ex

```diff
@@ -67,24 +67,34 @@ end
 
 defmodule MBTAV3API.Repository.Impl do
   @behaviour MBTAV3API.Repository
+
+  use Nebulex.Caching.Decorators
+
   alias MBTAV3API.JsonApi
 
+  @ttl :timer.hours(1)
+
   @impl true
   def alerts(params, opts \\ []), do: all(MBTAV3API.Alert, params, opts)
 
   @impl true
+  @decorate cacheable(cache: MBTAV3API.RepositoryCache, on_error: :nothing, opts: [ttl: @ttl])
   def route_patterns(params, opts \\ []), do: all(MBTAV3API.RoutePattern, params, opts)
 
   @impl true
+  @decorate cacheable(cache: MBTAV3API.RepositoryCache, on_error: :nothing, opts: [ttl: @ttl])
   def routes(params, opts \\ []), do: all(MBTAV3API.Route, params, opts)
 
   @impl true
+  @decorate cacheable(cache: MBTAV3API.RepositoryCache, on_error: :nothing, opts: [ttl: @ttl])
   def schedules(params, opts \\ []), do: all(MBTAV3API.Schedule, params, opts)
 
   @impl true
+  @decorate cacheable(cache: MBTAV3API.RepositoryCache, on_error: :nothing, opts: [ttl: @ttl])
   def stops(params, opts \\ []), do: all(MBTAV3API.Stop, params, opts)
 
   @impl true
+  @decorate cacheable(cache: MBTAV3API.RepositoryCache, on_error: :nothing, opts: [ttl: @ttl])
   def trips(params, opts \\ []), do: all(MBTAV3API.Trip, params, opts)
 
   @spec all(module(), JsonApi.Params.t(), Keyword.t()) ::
```

Comment:
Would it make more sense to only `@decorate cacheable` on `all/3` than on almost all of the individual resources separately?

Reply:
Good point!

Reply:
Do we want to cache alerts though? I can see it potentially being important to _not_ cache them.

Reply:
The alerts fn wasn't actually used since we use the Alerts stream instead - removed in [83e238b](https://github.com/mbta/mobile_app_backend/pull/231/commits/83e238bc328957767c9de51de2f0d9f11c4fef27)